client/engine/engine.go

```diff
@@ -132,6 +125,11 @@ func (e *Engine) handleAPIEvent(apiEvent APIEvent) {
 }
 
 // executeSideEffects executes the SideEffects declared by cranking an Objective
-func (e *Engine) executeSideEffects(protocols.SideEffects) {
-	// TODO
+func (e *Engine) executeSideEffects(sideEffects protocols.SideEffects) {
+	for _, message := range sideEffects.MessagesToSend {
+		e.toMsg <- message
+	}
+	for _, tx := range sideEffects.TransactionsToSubmit {
+		e.toChain <- tx
+	}
```

Comment:
This is the nub of the changeset: the engine now has outbound channels which are connected to services that run in their own goroutine.

Reply:
Useful comment, thanks!